core/src/components/textarea/textarea.scss

```diff
@@ -170,7 +170,8 @@
   word-break: break-word;
 }
 
-.native-textarea[disabled]:not(.cloned-input) {
+:host(.legacy-textarea) .native-textarea[disabled],
```

Comment:
We originally had the `:not(.cloned-input)` selector from this PR: https://github.com/ionic-team/ionic-framework/pull/26244.

Would that still apply here?

Reply:
That select does not matter because we still override the opacity here: https://github.com/ionic-team/ionic-framework/blob/feature-7.0/core/src/components/textarea/textarea.scss#L165